test/groovy/org/ods/usecase/DocGenUseCaseSpec.groovy

```diff
@@ -220,6 +220,60 @@ class DocGenUseCaseSpec extends SpecHelper {
         1 * nexus.storeArtifact(*_) >> nexusUri
     }
 
+    def "create document with watermark"() {
```

Comment:
Like the test a lot, but we need to do the same for the overall document.

Reply:
Added spec